code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -453,6 +453,106 @@
 					if(prob(10))
 						to_chat(H, "<span class='userdanger'>Being in the presence of [holder]'s [src] is interfering with your powers!</span>")
 
+/obj/item/weapon/nullrod/salt
+	name = "Holy Salt"
+	icon = 'icons/obj/food/containers.dmi'
+	icon_state = "saltshakersmall"
+	item_state = null
+	desc = "While commonly used to repel some ghosts, it appears others are downright attracted to it."
+	force = 0
+	throwforce = 0
+	var/ghostcall = 0
+
+
+/obj/item/weapon/nullrod/salt/attack(mob/living/carbon/M as mob, mob/living/carbon/user as mob)
+	if(!iscarbon(M))
+		return ..()
+
+	if(!user.mind || user.mind.assigned_role != "Chaplain")
+		to_chat(user, "<span class='notice'>You are not close enough with [ticker.Bible_deity_name] to use [src].</span>")
+		return
+
+	if(ghostcall)
+		to_chat(user, "<span class='notice'>You are already using [src].</span>")
+
+	user.visible_message("<span class='info'>[user] kneels[M == user ? null : " next to [M]"] and begins to utter a prayer to [ticker.Bible_deity_name] while drawing a circle with salt!.</span>", \
+		"<span class='info'>You kneel[M == user ? null : " next to [M]"] and begin a prayer to [ticker.Bible_deity_name] while drawing a circle!.</span>")
+
+	ghostcall = 1
+	if(do_after(user, 50, target = M))
+		notify_ghosts("The Chaplain is calling ghosts to [get_area(src)] with [name]!", source = src)
```

Comment:
To be honest this probably doesn't need a `do_after`; just a cooldown of something like 5 minutes, otherwise, a Chaplain can spam this on ghosts literally every 5 seconds--that'll get old fast.